cross/toolchain.cmake

```diff
@@ -31,6 +31,10 @@ else()
   message(FATAL_ERROR "Arch is ${TARGET_ARCH_NAME}. Only armel, arm, arm64 and x86 are supported!")
 endif()
 
+if(DEFINED ENV{TOOLCHAIN})
```

Comment:
I did cross-compile to ARM64 with aarch64-linux-gnu toolchain and made necessary changes in the build script for GCC to handle this. This is my main project. Since I tested this, I'd like to keep it if you don't mind.

Reply:
Oh, really? I am very surprised it fully worked. Ok, in such case, I take my comments back